packages/@sanity/desk-tool/src/panes/document/DocumentPane.tsx

```diff
@@ -1,4 +1,5 @@
-import {useDocumentType} from '@sanity/base/hooks'
+// eslint-disable-next-line camelcase
+import {unstable_useTemplatePermissions, useDocumentType} from '@sanity/base/hooks'
```

Comment:
i like renaming the import without the unstable so the linter doesn't go off at each new line

Reply:
I'm a bit torn on removing the `unstable_` prefix on imports like these. I think it's nice to be able to see clearly that it's unstable at the usage site as well. Perhaps we could modify the eslint rule to dirsegard `unstable_` instead?

Reply:
pushed a commit that allows `unstable_`-prefixes here: 95449799af